Dockerfile.assisted-installer

```diff
@@ -1,4 +1,8 @@
-FROM registry.ci.openshift.org/openshift/release:golang-1.19 AS builder
+FROM registry.access.redhat.com/ubi9/go-toolset:1.18 AS builder
```

Comment:
why are you downgrading the Go version here?
is it by mistake ?

Reply:
This is the actual go version, look at `go.mod`